src/julia-parser.scm

```diff
@@ -865,7 +865,7 @@
            (begin
              #;(if (and (number? ex) (= ex 0))
                  (error "juxtaposition with literal \"0\""))
-             `(call * ,ex ,(parse-unary s))))
+             `(call .* ,ex ,(parse-unary s))))
```

Comment:
`*` has different semantics from `.*`. I think it is a mistake to treat the latter as a superset of the former, as is being done with this implicit multiplication lowering to `.*`. Intuitively, `2x` means `2 * x`, not `2 .* x`.